S/Skia/build_tarballs.jl

```diff
@@ -0,0 +1,101 @@
+
+]
+
+
+
+# Bash recipe for building across all platforms
+script = raw"""
+
+
+cd $WORKSPACE/srcdir/
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd skia
+
+install_license LICENSE 
+
+bin/fetch-gn 
+python3 tools/git-sync-deps || true
+
+cp ../cskia/capi/sk_capi.cpp src/base/
+cp ../cskia/capi/sk_capi.h src/base/
+
+if [[ "${target}" == x86_64-* ]]; then
+bin/gn gen out/Static --args='target_cpu="x64" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+fi
+
+if [[ "${target}" == aarch64-* ]]; then
+bin/gn gen out/Static --args='target_cpu="arm64" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+
+fi
+
+if [[ "${target}" == riscv64-* ]]; then
+bin/gn gen out/Static --args='target_cpu="riscv" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+fi
+
+if [[ "${target}" == powerpc64le-* ]]; then
+bin/gn gen out/Static --args='target_cpu="powerpc64le" cc="clang" cxx="clang++" is_official_build=true skia_use_system_libjpeg_turbo=false skia_use_system_libpng=false skia_use_system_libwebp=false skia_use_system_zlib=false skia_use_vulkan=true skia_use_system_freetype2=false skia_use_fontconfig=true skia_enable_pdf=true  skia_use_system_icu=false skia_use_system_expat=false skia_use_harfbuzz=false skia_use_vulkan=true skia_use_gl=true extra_cflags=["-fpic", "-fvisibility=default"]'
+fi
+
+
+ninja -j${nproc} -C out/Static
+
+cd out/Static/
+
+clang++ -shared -o libskia.so   -Wl,--whole-archive libskia.a -Wl,--no-whole-archive libfreetype2.a libjpeg.a libbentleyottmann.a libcompression_utils_portable.a libdng_sdk.a libjsonreader.a libpathkit.a  libpiex.a libpng.a libskcms.a libsksg.a libskshaper.a libskunicode_core.a libskunicode_icu.a libsvg.a -fpic -fvisibility=default -lstdc++ -dl -lfontconfig -lGL 
+
+mkdir -p ${prefix}/lib/
+cp libskia.so ${prefix}/lib/
```

Comment:
```suggestion
install -Dvm 755 "libskia.${dlext}" "${libdir}/libskia.${dlext}"
```
Cleaner command, and it ensures permissions are correct.